crates/precompile/Cargo.toml

```diff
@@ -32,6 +32,12 @@ secp256k1 = { version = "0.28.2", default-features = false, features = [
     "rand",
     "global-context",
 ], optional = true }
+cometbft = { version = "=0.1.0-alpha.2" }
```

Comment:
Is there any beta or stable version?

Reply:
no.. they only have 1 version. https://github.com/cometbft/cometbft-rs/tags

do we need to fork this repo to bnb-chain org?